openmc/cell.py

```diff
@@ -79,13 +79,12 @@ class Cell(object):
     translation : Iterable of float
         If the cell is filled with a universe, this array specifies a vector
         that is used to translate (shift) the universe.
-    offsets : ndarray
-        Array of offsets used for distributed cell searches
-    distribcell_index : int
-        Index of this cell in distribcell arrays
-    distribcell_paths : list of str
-        The paths traversed through the CSG tree to reach each distribcell
-        instance
+    paths : list of str
+        The paths traversed through the CSG tree to reach each cell
+        instance. This property is initialized by calling the
+        :meth:`Geometry.determine_paths` method.
```

Comment:
Oh, right, oops. I guess I'll change it to say that paths are determine when a root universe is assigned to a `Geometry`.